### PR TITLE
[WIP][Build] Add RIDs of Tizen.6.0.0 and Set RIDs to vconf automatically

### PR DESCRIPTION
**Pull request by JongHeonChoi**
### API Changes ###

 - ACR:

### CODE REVIEW

**Comment by WonyoungChoi** [comment]
@JongHeonChoi makespec.sh is executed by TizenFX CI/CD scripts on the docker image "https://hub.docker.com/r/tizendotnet/tizenfx-build-worker". So the `jq` should be installed to the docker image first. And we can consider other approach to rewrite the makespec.sh script with Python. It can be written without any other dependent tools.

**Comment by JongHeonChoi** [comment]
@WonyoungChoi Anyway, the way I thought it would be to parse the value of 'pkg/Tizen.NET/runtime.json' in TizenFX. Also, it is to set the vconf in TizenFX. Then, the launcher can automatically set the native library path by reading the vconf value(rid).

File: packaging/makespec.sh [hunk 1]
```diff
@@ -24,3 +24,20 @@ echo "# Auto-generated from $(basename $RPMSPEC_IN) by makespec.sh" | cat - $RPM
 sed -i -e "s/@api_version@/$API_VERSION/g" $RPMSPEC
 sed -i -e "s/@rpm_version@/$RPM_VERSION/g" $RPMSPEC
 sed -i -e "s/@nuget_version@/$NUGET_VERSION/g" $RPMSPEC
+
```
Comment (HJLeee) [inline packaging/makespec.sh]: I think @WonyoungChoi wants rewritting this part to a separate python script and calling it here instead installing `jq`.
Am I correct?

Comment (JongHeonChoi) [reply packaging/makespec.sh]: Created a separate Python script and modified it to call it.

**Comment by JongHeonChoi** [comment]
@WonyoungChoi 
Does API7 support tizen.6.0.0 rid?

**Comment by WonyoungChoi** [comment]
@JongHeonChoi No, API7 will be installed to Tizen 5.5 M3.

**Comment by JongHeonChoi** [comment]
| API Level | Target Framework | Platform | RIDs |
|:---:|:---:|:---:|:---:|
| API8 | tizen80 | Tizen 6.0 | 6.0.0 |
| API7 | tizen70 | Tizen 5.5 M3 | 5.5.0 |
| API6 | tizen60 | Tizen 5.5 M2 | 5.5.0 |
| API5 | tizen50 | Tizen 5.0 | 5.0.0 |
| API4 | tizen40 | Tizen 4.0 | 4.0.0 |

**Comment by WonyoungChoi** [comment]
LGTM 👍

**Pull request merged**